File: alphabeta pruning tictactoe/src/TicTacToe/AI.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package TicTacToe;
import Inteligence.*;
/**
 *
 * @author Taufiq
 */
public class AI extends Player{
    public AI(String PlayerName){
        super(PlayerName);
        
    }
    public int NextMove(Board gBoard){
        Inteligence intel= new Inteligence(gBoard);
        return intel.Move();
    }

}

File: mentalExercise/Alpha beta pruning and tictactoe/AI.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Alpha_beta_pruning_and_tictactoe
{
    public class AI: Player{
    public AI(String PlayerName): base(PlayerName){}
    public int NextMove(Board gBoard){
        Inteligence intel= new Inteligence(gBoard);
        return intel.Move();
    }

}
}
